### PR TITLE
Messenger event log

### PR DESCRIPTION
**Pull request by rmheuer**
Adds an event log to the Messenger tool so you can see when other clients connect and disconnect from Messenger.

### CODE REVIEW

File: Messenger/MessengerClient/src/main/java/com/swrobotics/messenger/client/MessengerClient.java [hunk 1]
```diff
@@ -123,7 +125,15 @@ private void startConnectThread() {
...
-                    for (String listen : new HashSet<>(listening)) {
+                    // Prioritize listening to EVENT_TYPE so that we receive our own events
+                    // when listening to the rest of them
+                    Set<String> listeningCopy = new HashSet<>(listening);
+                    if (listeningCopy.contains(EVENT_TYPE))
+                        listen(EVENT_TYPE);
+
+                    for (String listen : listeningCopy) {
+                        if (listen.equals(EVENT_TYPE))
+                            continue;
```
Comment (foehammer127) [inline Messenger/MessengerClient/src/main/java/com/swrobotics/messenger/client/MessengerClient.java:136]: Pls Fix Indentation Here (Just a coding style thing.

File: ShuffleLog/src/main/java/com/swrobotics/shufflelog/tool/messenger/MessengerTool.java [hunk 2]
```diff
@@ -53,8 +75,7 @@ private void showConnectionParams() {
             ImGui.tableNextColumn();
             ImGui.text("Status:");
             ImGui.tableNextColumn();
-            boolean connected = msg.isConnected();
-            if (connected) {
```
Comment (foehammer127) [inline ShuffleLog/src/main/java/com/swrobotics/shufflelog/tool/messenger/MessengerTool.java:57]: Bruh

**Review by foehammer127 — COMMENTED** [review]
Banger PR as always

File: ShuffleLog/src/main/java/com/swrobotics/shufflelog/tool/messenger/MessengerTool.java (hunk 2 above)
Comment (rmheuer) [reply ShuffleLog/src/main/java/com/swrobotics/shufflelog/tool/messenger/MessengerTool.java:57]: Yep that's fixed now

File: Messenger/MessengerClient/src/main/java/com/swrobotics/messenger/client/MessengerClient.java (hunk 1 above)
Comment (rmheuer) [reply Messenger/MessengerClient/src/main/java/com/swrobotics/messenger/client/MessengerClient.java:136]: I don't see anything wrong with the indentation here, which line(s) are incorrect?

**Pull request merged**